snappy_pipeline/workflows/cbioportal_export/model.py

```diff
@@ -150,8 +162,6 @@ class CbioportalExport(SnappyStepModel):
     Works together with filter_set, ignored when "filter_list" is selected
     """
 
-    filter_list: list[Filter] = []
-
```

Comment:
Perhaps I haven't spotted it, but where did this go? If filter_set is deprecated and points to filter_list, then why is filter_list removed but filter_set remains?

Reply:
Because in the filtration step, when `sets` is selected, all filters are computed (regardless of the choice of set), so the user could select another one for cBioPortal upload.
I put a comment about that in the model.

Reply:
Ah I see